File: arr.lua
#!/usr/bin/luajit
local size = 64
local array = {}

math.randomseed(os.time())

for i = 1, size do
    array[i] = math.random(0, 20)
end

io.write("Random number array:")
for i = 1, size do
    io.write(" " .. array[i])
end
io.write("\n\n")

File: arr.c
#!/usr/bin/tcc -run
#include <stdio.h>
#include <time.h>
#include <stdlib.h>

#define size 64

void main() {
    int array[size];
    int i;
    srand(time(NULL));

    for (i = 0; i < size; i++) {
        array[i] = rand() % 20;
    }

    printf("Random number array:");
    for (i = 0; i < size; i++) {
        printf(" %d", array[i]);
    }
    printf("\n\n");
}